cluster-operator/src/main/java/io/strimzi/operator/cluster/model/cruisecontrol/BrokerCapacity.java

```diff
@@ -21,7 +21,10 @@ public class BrokerCapacity {
     public static final String DEFAULT_OUTBOUND_NETWORK_CAPACITY_IN_KIB_PER_SECOND = "10000";
 
     private static final String DEFAULT_BROKER_DOC = "This is the default capacity. Capacity unit used for disk is in MiB, cpu is in number of cores, network throughput is in KiB.";
-    protected static final String DEFAULT_CPU_CORE_CAPACITY = "1";
+    /**
+     * Default cpu core capacity
+     */
+    public static final String DEFAULT_CPU_CORE_CAPACITY = "1.0";
```

Comment:
Why does this need to be public now? It looks to me like it doesn't even need to be `protected` but `/* test*/ would be sufficient.

Reply:
As CruiseControlTest is in a different package to this, it needs to be public to access it from CruiseControlTest.